### ISSUE TITLE
Choosing the appropriate image contrast per task

### ISSUE THREAD

**Issue by jcohenadad** [sct-pipeline/critical-ms-lesion#2]
Based on the data @plbenveniste and I received from the HC, we can already make decisions about what image contrast to choose for each task. Below is a proposal:

- SC segmentation (contrast-agnostic)
  - `Csp Ax T2`
  - `Csp Sag 3D Fiesta`
  - `Csp Sag Cube T2 Focus`
- vertebral labeling (TotalSpineSeg)
  - `ORIG SAG T2 LOC UPPER DL`
  - also consider other contrasts used for SC seg if they are "reliable enough"
- lesion segmentation (PL's model)
  - `Sag T2 STIR`
  - `Ax T2`
  - `Ax MERGE`

**Comment by plbenveniste** [comment]
From your comment, I understand that we should use the original file only for vertebral labeling. For the rest, we should use the post-processed file right ? 
Also, for the lesion segmentation, do you want to include only cervical spine ? Or thoracic and lumbar as well (in this case, it would make sense to include thoracic and lumbar in the previous sections as well) ?

**Comment by jcohenadad** [comment]
> From your comment, I understand that we should use the original file only for vertebral labeling. For the rest, we should use the post-processed file right ?

not necessarily--  we could go with the denoised one (if this is what you meant by "post-processed")--
Also, we should consider the other contrasts-- i've updated my initial post

> Also, for the lesion segmentation, do you want to include only cervical spine ? Or thoracic and lumbar as well (in this case, it would make sense to include thoracic and lumbar in the previous sections as well) ?

let's start with cervical only, for the sake of simplicity-- I suspect there is little amount of lesions in the thoraco-lumbar-- but as we receive the data we can reconsider
